docs/build/build-protocol-info.md

```diff
@@ -132,7 +132,7 @@ More info:
 
 - [Lockable Currency](https://paritytech.github.io/substrate/master/frame_support/traits/trait.LockableCurrency.html)
 - [Lock Withdraw Reasons](https://paritytech.github.io/substrate/master/frame_support/traits/struct.WithdrawReasons.html)
-- [Vesting Info](https://paritytech.github.io/substrate/master/pallet_vesting/struct.VestingInfo.html)
+- [Vesting](https://paritytech.github.io/substrate/master/pallet_vesting/index.html)
```

Comment:
https://paritytech.github.io/substrate/master/pallet_vesting/struct.VestingInfo.html is correct and functional

Reply:
Thanks for the review @NukeManDan!  I didn't see you were working on #3437 when I was working on this.  After pulling in your changes I noticed we had many updated urls that matched but I also had to resolved a few differences.  I must have overwrote this one accidentally with what I had. :+1:

Reply:
Also thanks for updating all the `docs.substrate.io` links!  That was next on my list but after your PR all the remaining urls appear valid and are no longer redirecting.